hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneManagerVersion.java

```diff
@@ -40,6 +40,8 @@ public enum OzoneManagerVersion implements ComponentVersion {
   LIGHTWEIGHT_LIST_KEYS(4, "OzoneManager version that supports lightweight"
       + " listKeys API."),
 
+  ATOMIC_REWRITE_KEY(5, "OzoneManager version that supports rewriting key as atomic operation"),
```

Comment:
We'll need to update the specific version number later, as 5 is being added for HDDS-10435.